src/status_im/contexts/settings/wallet/keypairs_and_accounts/remove/view.cljs

```diff
@@ -0,0 +1,38 @@
+(ns status-im.contexts.settings.wallet.keypairs-and-accounts.remove.view
+  (:require [quo.core :as quo]
+            [quo.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [status-im.common.standard-authentication.core :as standard-auth]
+            [status-im.contexts.settings.wallet.keypairs-and-accounts.remove.style :as style]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn view
+  [{:keys [name key-uid]}]
+  (let [on-remove (rn/use-callback #(rf/dispatch
+                                     [:wallet/remove-keypair key-uid])
+                                   [key-uid])]
+    [:<>
+     [quo/page-top
+      {:container-style style/header-container
+       :title           (i18n/label :t/remove-key-pair-and-derived-accounts)
+       :title-size      :heading-2
```

Comment:
If the style of the component in the design system and the screen differs, it's best to check with the Design team to confirm it.